pages/servers.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: page
+title:  "Servers"
+permalink: /servers/
+---
+
+List of servers (in alphabetical order) that provide a CCTray feed:
+
+### Buddybuild
+
+* Server home: <https://www.buddybuild.com/>
+* CCMenu setup instructions: <http://docs.buddybuild.com/docs/ccmenu>
+
+### CircleCI
+
+* Server home: <https://circleci.com/>
+* CCMenu setup instructions: <https://circleci.com/docs/polling-project-status>
+
+### CruiseControl
+
+* Server home: <http://cruisecontrol.sourceforge.net/>
+* Default feed location (classic): `/xml`
+* Default feed location (dashboard): `/cctray.xml` or `/dashboard/cctray.xml`
+* CCMenu autodetect feed URL: supported
+
+### CruiseControl.NET
+
+* Server home: <http://www.cruisecontrolnet.org/>
+* Default feed location: `/XmlStatusReport.aspx` or `/ccnet/XmlStatusReport.aspx`
+* CCMenu autodetect feed URL: supported
+
+### CruiseControl.rb
+
+* Server home: <http://cruisecontrolrb.thoughtworks.com/>
+* Default feed location: `/XmlStatusReport.aspx` or `/ccnet/XmlStatusReport.aspx`
+* CCMenu autodetect feed URL: supported
+
+### Go
+
+* Server home: <http://www.go.cd/>
+* Default feed location: `/go/cctray.xml`
+* CCMenu autodetect feed URL: supported
+
+### GreenhouseCI
+
+* Server home: <http://greenhouseci.com/>
+* CCMenu setup instructions: <http://blog.greenhouseci.com/greenhouse/update/ccmenu-support/>
+
+### Hudson
+
+* Server home: <http://hudson-ci.org/>
+* Default feed location: `/cc.xml` or `/hudson/cc.xml`
+* CCMenu autodetect feed URL: supported
+
+### Jenkins
+
+* Server home: <http://jenkins-ci.org/>
+* Default feed location: `/cc.xml` or `/hudson/cc.xml`
+* CCMenu autodetect feed URL: supported
+* Note: If you have problems with authentication, please check that you have CCMenu 1.9 or newer.
+
+### Semaphore
+
+* Server home: <https://semaphoreapp.com/>
+* CCMenu setup instructions: <https://github.com/renderedtext/semaphore-docs-new/blob/master/source/docs/cctry.md>
+
+### Snap CI
```

Comment:
Snap CI no longer exists so shouldn't be included 😢

Reply:
Will remove.